lib/job.js

```diff
@@ -14,7 +14,13 @@ var Job = module.exports = function Job(args) {
   delete args.agenda;
 
   // Process args
-  args.priority = parsePriority(args.priority) || 0;
+
+  // allow priority overrides (e.g. `job.now('foo', { priority: 'low' })`)
```

Comment:
Do you have any thoughts on how best to rewrite my tests to for priority and FIFO?  That was my main use case (besides the issue on GH) for adding this.